application/classes/Ushahidi/Repository/Form/Attribute.php

```diff
@@ -233,6 +233,25 @@ public function getByForm($form_id)
 		return $this->getCollection($results->as_array());
 	}
 
+    /**
+     * Filter the form ids that are included in the attributes the user selected for a CSV
+     * @param $include_attributes
+     * @return null|array
+     */
+    public function getFormsByAttributes($include_attributes) {
+	    if ($include_attributes && !empty($include_attributes)) {
```

Comment:
Does this need both checks, would empty not suffice?

Reply:
oh, that was silly. Yep, only isempty is needed. I'll fix